packages/build/README.md

```diff
@@ -0,0 +1,50 @@
+# @loopback/build
+
+The module contains a set of common scripts and default configurations to
+build LoopBack 4 modules, including:
+
+- lb-tsc: Use `tsc` to compile typescript files
+- lb-dist: Detect the correct distribution target: `dist` => ES2017, `dist6` => ES2015
+- lb-tslint: Run `tslint` with `<project>/tslint.build.json`
+- lb-prettier: Run `prettier`
+
+# Usage
+
+To use `@loopback/build` for your package:
+
+1. Run the following command to add `@loopback/build` as a dev dependency.
+
+`npm i @loopback/build --save-dev`
+
+2. Configure your project package.json as follows:
+```json
+"scripts": {
+    "acceptance": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
+    "build": "npm run build:dist && npm run build:dist6",
+    "build:current": "lb-tsc",
+    "build:dist": "lb-tsc es2017",
+    "build:dist6": "lb-tsc es2015",
+    "build:apidocs": "lb-apidocs",
+    "tslint": "lb-tslint",
+    "tslint:fix": "npm run tslint -- --fix",
+    "prettier:cli": "lb-prettier \"**/*.ts\"",
```

Comment:
This is cool if the project uses the same prettier config as we do. However, if the project wants to customize their settings, how can they do that? Could you please explain that in README?

Reply:
See README